### PR TITLE
Add dir storybook decorator

### LINKED ISSUE #469: Storybook: update to latest inputProvider; remove rtl-addon
~~Blocked on #468~~

**Is your feature request related to a problem? Please describe.**
A new & improved version of inputProvider will be released in `@bbc/psammead-storybook-helpers@2.0.0` in #468. This will make it easier to preview components in different languages, with the ability to pass in the correct text direction and script values. Using this helper function will also make `storybook-addon-rtl` obsolete, so that should be removed as part of this issue.

**Describe the solution you'd like**
Add a decorator for stories which do not need an inputProvider for the text previewing functionality and only the direction control functionality as follows:
`const DirDecorator = storyFn => inputProvider(null, () => {storyFn()});`

**Testing notes**
[Tester to complete]

Dev insight: Check storybook pages render and have expected functionality.

**Additional context**

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by ghost**
Resolves #469 

**Overall change:** Implement direction storybook decorator and remove the `storybook-addon-rtl` dependency

**Code changes:**

- Implemented `dirDecorator` storybook decorator helper function

**Note to testers**
- Arabic, Pashto, Persian and Urdu are `RTL` while the rest `LTR` languages.
---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by dr3 — CHANGES_REQUESTED** [review]
Sorry if we didn't mention this, You cant bump a package and use it in the same PR, as you cant install it from the registry until it's published.

You will need to split the change for `psammead-storybook-helpers` into its own PR, get it reviewed and tested and merged, then install it in each of the other packages in a separate PR. This PR only works with `npm run install:packages:link`, it should with `npm run ci:packages`

**Comment by ghost** [comment]
Thanks, I wasn't aware of that. I will create a separate issue to make the `Brand`, `Timestamp` and `MediaIndicator` to use the `dirDecorator`

File: .storybook/addons.js [hunk 1]
```diff
@@ -1,3 +1,2 @@
 import '@storybook/addon-notes/register';
 import '@storybook/addon-knobs/register';
-import 'storybook-addon-rtl/register';
```
Comment (dr3) [inline .storybook/addons.js]: We shouldn't remove these until we use this package elsewhere :)

File: packages/utilities/psammead-storybook-helpers/README.md [hunk 2]
```diff
@@ -72,6 +76,28 @@ storiesOf('Caption', module)
...
+import { dirDecorator } from '@bbc/psammead-storybook-helpers';
+import Brand from '@bbc/psammead-brand';
+import { news as brandSVG } from '@bbc/psammead-assets';
+
+storiesOf('Brand', module)
+  .addDecorator(withKnobs)
+  .addDecorator(dirDecorator)
+  .add(
+    'default',
+    (inputValues, script, dir) => (
```
Comment (dr3) [inline packages/utilities/psammead-storybook-helpers/README.md]: This example doesnt need `(inputValues, script, dir) => (`

Can you make this a really simple example please, just with `<h1>Lorem Ipsum</h1>` please

File: packages/utilities/psammead-storybook-helpers/src/decorators.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import inputProvider from './input-provider';
+
+/* eslint-disable import/prefer-default-export */
+export const dirDecorator = storyFn => {
```
Comment (dr3) [inline packages/utilities/psammead-storybook-helpers/src/decorators.js:4]: Just like how we have https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-storybook-helpers/src/index.stories.jsx which shows off the input provider, can you make a simple example of the dirDecorator please in a real stories.jsx file

**Review by dr3 — APPROVED** [review]

**Review by dr3 — CHANGES_REQUESTED** [review]
Pressed wrong button, apologies

File: .storybook/addons.js (hunk 1 above)
Comment (ghost) [reply .storybook/addons.js]: Does it also mean that we shouldn't change the config?

Comment (dr3) [reply .storybook/addons.js]: We shouldnt remove `storybook-addon-rtl` untill we use the new dirDecorator everywhere

File: packages/utilities/psammead-storybook-helpers/README.md [hunk 4]
```diff
@@ -13,10 +13,13 @@ This package provides a collection of common values that are used in storybook b
     - `name`: String uniquely identifying this slot in the story. Required.
     - `defaultText`: String to use when the story is showing English text. Optional.
 - `renderFn`: `function(slotTexts, script, dir)` Required.
+
```
Comment (dr3) [inline packages/utilities/psammead-storybook-helpers/README.md]: new line?

File: packages/utilities/psammead-storybook-helpers/README.md [hunk 5]
```diff
@@ -28,16 +30,19 @@ npm install @bbc/psammead-storybook-helpers --save-dev
...
 
 const label = 'Languages';
 const defaultValue = 'This is a caption';
 const groupIdentifier = 'CAPTION VARIANTS';
 
 <Caption>
-  {select(label, LANGUAGE_VARIANTS, LANGUAGE_VARIANTS.english, groupIdentifier).text}
-</Caption>
+  {
+    select(label, LANGUAGE_VARIANTS, LANGUAGE_VARIANTS.english, groupIdentifier)
```
Comment (dr3) [inline packages/utilities/psammead-storybook-helpers/README.md]: Can you revert this line please?

File: packages/utilities/psammead-storybook-helpers/README.md [hunk 6]
```diff
@@ -72,6 +75,22 @@ storiesOf('Caption', module)
...
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import { dirDecorator } from '@bbc/psammead-storybook-helpers';
+
+storiesOf('Example', module)
+  .addDecorator(withKnobs)
+  .addDecorator(dirDecorator)
+  .add('default', () => <h1>Lorem Ipsum</h1>, {
+    knobs: { escapeHTML: false },
```
Comment (dr3) [inline packages/utilities/psammead-storybook-helpers/README.md]: Does this example need the knobs value?

Comment (ghost) [reply packages/utilities/psammead-storybook-helpers/README.md]: no i doesn't

File: packages/utilities/psammead-storybook-helpers/package.json [hunk 7]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-storybook-helpers",
-  "version": "2.0.0",
+  "version": "2.0.1",
```
Comment (dr3) [inline packages/utilities/psammead-storybook-helpers/package.json]: Can this be `2.1.0` please as it ['adds functionality in a backwards-compatible manner'](https://semver.org/) :)

**Review by dr3 — CHANGES_REQUESTED** [review]

File: packages/utilities/psammead-storybook-helpers/src/index.stories.jsx [hunk 8]
```diff
@@ -35,4 +35,8 @@ storiesOf('Input Provider', module)
       ),
     ),
     { notes, knobs: { escapeHTML: false } },
-  );
+  )
+  .addDecorator(dirDecorator)
```
Comment (dr3) [inline packages/utilities/psammead-storybook-helpers/src/index.stories.jsx:42]: image

You may need to split these up, like I have [here](https://github.com/bbc/simorgh/blob/ab5ddf1f30dc01e575497b3369eb11178886449b/src/app/containers/StoryPromo/index.stories.jsx#L20-L26)

**Review by dr3 — CHANGES_REQUESTED** [review]

**Review by dr3 — DISMISSED** [review]

File: packages/utilities/psammead-storybook-helpers/README.md [hunk 9]
```diff
@@ -17,6 +17,8 @@ This package provides a collection of common values that are used in storybook b
   - `script`: A [script](https://github.com/bbc/psammead/tree/latest/packages/utilities/gel-foundations#script-support) corresponding to the language selected by the storybook user.
   - `dir`: Either `'ltr'` or `'rtl'`, corresponding to the language currently selected by the storybook user.
 
+`dirDecorator` - A storybook decorator function that uses `inputProvider` internaly to provide direction control only
```
Comment (DenisHdz) [inline packages/utilities/psammead-storybook-helpers/README.md]:
```suggestion
`dirDecorator` - A storybook decorator function that uses `inputProvider` internally to provide direction control only
```

Comment (DenisHdz) [reply packages/utilities/psammead-storybook-helpers/README.md]: Just a little typo

Comment (ghost) [reply packages/utilities/psammead-storybook-helpers/README.md]: Thanks. Fixed

**Review by andrew-nowak — APPROVED** [review]
lgtm :+1:

**Review by dr3 — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM. Happy for this to be merged

**Pull request merged**